packages/providers/upload-aws-s3/lib/index.js

```diff
@@ -21,15 +22,18 @@ module.exports = {
       ...config,
     });
 
+    const ACL = get(['params', 'ACL'], config, 'public-read');
```

Comment:
Thank you for the QA @innerdvations !

We changed the lodash get to use `lodash/fp` and it seems it does not have the 'default' argument. I have created a pr for it!

> The "copy link" in the content manager for a related media item in a private bucket includes the signed url parameters. I don't know if that's intended, but since I understood it as those parameters only being for the viewing of the image within the admin ui I wasn't sure if that link should only be the raw image url.

Good catch! We originally did not think bout this, but I think we could leave it this way, as it will provide users a way to share the image, which will expire in a certain amount of time.

Reply:
Sounds good. I wonder if we should document that the links will only last that long? I could see content editors attempting to use that link somewhere in their content.

Reply:
That's a good point indeed, this is a use case that we will debate in the team on further iterations.